esti/import_test.go

```diff
@@ -288,6 +296,200 @@ func TestAzureDataLakeV2(t *testing.T) {
 	}
 }
 
+func TestImportNew(t *testing.T) {
+	blockstoreType, importPath, expectedContentLength := setupByBlockstoreType(t)
+
+	ctx, _, repoName := setupTest(t)
```

Comment:
Why does it matter? Each test case imports to a different branch

Reply:
- Having a clean start. Working on existing repository can be bad/good if it can hide issues only if we start clean.
- Can use t.Parallel() to test faster if needed